.idea/Ligandr.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I would add `.idea` to your `.gitignore` file, so these Pycharm files don't get included in version control